pkg/cli/tail.go

```diff
@@ -242,6 +288,30 @@ func (c *TailCommand) tail(ctx context.Context) ([]*loggingpb.LogEntry, error) {
 	return ls, nil
 }
 
+func (c *TailCommand) StreamTail(ctx context.Context, logCh chan<- *loggingpb.LogEntry) error {
```

Comment:
We probably don't need this helper func? also tail helper func, and createPuller, it is hard to read with too many helper funcs and I don't see why we need them?

Reply:
Updated.